CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## Unreleased
+
+### Dependencies
+
+- Upgrade sample and E2E tests to RN 0.75 ([#4155](https://github.com/getsentry/sentry-react-native/pull/4155))
```

Comment:
I usually don't add changelog for changes that have no impact/changes to the clients. 

Also, the dependencies field is usually used to bump packages/libraries that users will update when using a new version of Sentry React Native, like a new version of Sentry Cocoa, Sentry Wizard, Sentry JavaScript,...

```suggestion
```